lib/TBDGen/TBDGen.cpp

```diff
@@ -479,6 +479,10 @@ void TBDGenVisitor::addConformances(const IterableDeclContext *IDC) {
   for (auto conformance : IDC->getLocalConformances(
                             ConformanceLookupKind::NonInherited)) {
     auto protocol = conformance->getProtocol();
+    if (Opts.PublicSymbolsOnly &&
+        getDeclLinkage(protocol) != FormalLinkage::PublicUnique)
```

Comment:
Do we need to also accept `FormalLinkage::PublicNonUnique` to to cover protocols imported from Objective-C?

Reply:
For this particular workaround, I don't think Objective-C protocols are relevant because the crash occurs specifically with internal protocols with associated types.
And other places inside the `TBDGenVisitor` also only accept `PublicUnique` so I'm trying to keep it concise and consistent.
We can add `PublicNonUnique` later if it's causing problems.